Content.Server/Cargo/CargoSystem.Console.cs

```diff
@@ -52,9 +51,16 @@ public sealed partial class CargoSystem
 
         private void InitializeConsole()
         {
+            SubscribeLocalEvent<CargoConsoleComponent, ComponentInit>(OnInit);
             SubscribeLocalEvent<RoundRestartCleanupEvent>(Reset);
             Reset();
         }
+        private void OnInit(EntityUid uid, CargoConsoleComponent console, ComponentInit args)
+        {
+            var receiver = EnsureComp<SignalTransmitterComponent>(uid);
+            if (!receiver.Outputs.ContainsKey("Order Sender"))
+                receiver.AddPort("Order Sender");
+        }
```

Comment:
IDK why the other signal code does this but you should put this on the prototype itself; at least make it an EnsureCompWarn.